src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/legacyMedia/index.js

```diff
@@ -1,11 +1,29 @@
-const generateVideoBlock = block => {
+import path from 'ramda/src/path';
+
+const getImageUrl = block => {
+  let imageURL = path(['image', 'href'], block);
+
+  if (imageURL) {
+    imageURL = imageURL.replace('http://', 'https://');
+  }
+
+  return imageURL;
+};
```

Comment:
Could be compressed to this as an option
```suggestion
const getImageUrl = block =>
  pathOr('', ['image', 'href'], block).replace('http://', 'https://');
```